src/content/posts/how-to-invest-using-angor/index.md

```diff
@@ -0,0 +1,78 @@
+
+- **Navigate to the Projects Section**: Once logged in, look for the 'Projects Browse' tab in the main menu and click on it. This will take you to the section where all available projects are listed.
+
+    ![List of project will be here](./assets/project-list.svg)
+
+- **Explore the Projects**
+1. **Review Details**: Each project listing includes a variety of details such as the project’s objectives, the team behind it, its funding goals, and timelines. Spend some time going through these details to understand the scope and purpose of each project.
+
+2. **Assess Goals**: Check the funding goals to see how much the project aims to raise. This will give you an idea of the scale of the project and its financial needs. The "Smart Farming Solutions" project has a target amount of 50 TBTC.
+3. **Understand Timelines**: Look at the project timelines to see the expected milestones and deadlines. This helps you gauge how long your investment will be tied up and when you can expect to see progress or returns. <br>For instance, the stages of the "Smart Farming Solutions" project include 10% completion by 30/04/2024, 30% by 20/05/2024, and 60% by 30/05/2024.
+
+    
+    ![Project Insights Overview](./assets/project-insider.svg)
+
+Taking the time to thoroughly explore the projects will help you make an informed decision about where to invest your funds.
+
+### Create a Wallet
+
+Before you can invest in any project, you need to set up a digital wallet on Angor.
+
+![How to invest using Angor](./assets/how-to-create-wallet.gif)
+
+- **Sign Up or Log In**:  If you’re new to Angor, you’ll need to create an account. If you already have an account, simply log in.
+
+    ![Create wallet view](./assets/create-wallet-screen.svg)
+
+- **Navigate to the Wallet Section**: Once logged in, find and click on the 'Wallet' section from your dashboard.
+
+- **Create Your Wallet**: Follow the on-screen instructions to create your digital wallet. This typically involves:
+
+    1. **Setting a Strong Password**: Choose a password that is secure to protect your wallet.
+    2. **Backing Up Recovery Phrases**: You’ll be given a set of recovery phrases. Make sure to store these securely, as they are essential for accessing your wallet if you forget your password.
+    3. **Understand Timelines**: Look at the project timelines to see the expected milestones and deadlines. This helps you gauge how long your investment will be tied up and when you can expect to see progress or returns. <br>For instance, the stages of the "Smart Farming Solutions" project include 10% completion by 30/04/2024, 30% by 20/05/2024, and 60% by 30/05/2024.
+    4. **Fund Your Wallet**: Transfer the necessary amount of cryptocurrency into your new wallet. This step is crucial, as you need funds in your wallet to invest in projects.
+
+        ![Wallet after creation](./assets/wallet-after-creation.svg)
+
+
+
+### Claim Back Funds with a Penalty
```

Comment:
you are missing the step of investing, you  need to tell how to actually invest, this process requires wating for the founder to approve the investment.

Reply:
I will look into that with a more detailed overview.

Reply:
@dangershony Now, it's looking clear.